src/App/Repository/PostRepository.php

```diff
@@ -56,6 +56,8 @@ class PostRepository extends OhanzeeRepository implements
     // Provides `postPermissions`
     use InteractsWithPostPermissions;
 
+    const UNPRIVILEGED_REQUEST_LIMIT = 20;
```

Comment:
Given we would always have limits  in place, we would just need different limits for privileged vs unprivileged.

Also, it would be good to have them configurable through a file in `config/` (and environment variables)

Reply:
Good idea, will look into that.

Reply:
done